.github/workflows/backend.yml

```diff
@@ -108,8 +108,12 @@ jobs:
         case:
           - name: cluster-test-mysql
```

Comment:
Can we change this to `cluster-test-mysql-with-zookeeper-registry` for better differentiation?

Reply:
Done